wallet/src/de/schildbach/wallet/service/BlockchainStateDataProvider.kt

```diff
@@ -193,6 +193,10 @@ class BlockchainStateDataProvider @Inject constructor(
         return syncStageFlow
     }
 
+    override fun getSyncStage(): PeerGroup.SyncStage {
+        return syncStageFlow.value ?: PeerGroup.SyncStage.OFFLINE
+    }
+
```

Comment:
Since `getSyncStage` isn't a suspend function and simply returns a value, it might be better to make it a val in the interface. Or even expose a `val syncStage: StateFlow` and let the caller decide to observe or get the value

Reply:
We could do this for the other items as well.  I will consider this in the next PR.